build.gradle

```diff
@@ -66,6 +66,11 @@ dependencies {
             'org.eclipse.jetty.websocket:websocket-jetty-api:' + jettyVersion,
             'org.eclipse.jetty.websocket:websocket-jetty-server:' + jettyVersion,
     )
+
+    // Websocket client libs
+    compileOnly 'jakarta.websocket:jakarta.websocket-client-api:2.2.0-M1'
+    implementation 'org.glassfish.tyrus.bundles:tyrus-standalone-client:2.2.0-M1'
```

Comment:
What does this dependency provide?

Reply:
It adds implementation for some jakarta classes, I've replaced it with the jersey one to make it more in line with the impl's we are using for our ws server